app/data.py

```diff
@@ -57,3 +57,10 @@ def drop_index(self, collection: str):
 
     def search(self, collection: str, user_search: str) -> List[Dict]:
         return self.read(collection, {"$text": {"$search": user_search}})
+
+    def scan_collections(self):
+        output = {}
+        cli = MongoClient(getenv("MONGO_URL"))[self.cluster]
```

Comment:
This is ok for a temporary solution. In the long run we should consider refactoring the MongoDB class such that we don't repeat `MongoClient(getenv("MONGO_URL"))[self.cluster]` in slightly different forms all over the place.